provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/client/PpmsClient.java

```diff
@@ -10,6 +12,17 @@
  * objects.
  */
 public interface PpmsClient {
+
+  CareSitesResponse careSitesByCity(String city);
+
+  LocationWrapper careSitesById(String id);
+
+  LocationWrapper careSitesByName(String name);
```

Comment:
It's confusing that some `careSites` methods return `CareSitesResponse` while others return `LocationWrapper`. Is there something missing from one of the sets of method names (provide services?) to indicate the differences in the data returned?